gruntwork-install

```diff
@@ -55,7 +55,7 @@ function log {
   local -r timestamp=$(date +"%Y-%m-%d %H:%M:%S")
   local -r scriptname="$(basename "$0")"
 
-  >&2 echo -e "${timestamp} [${level}] [$scriptname] ${message[@]}"
+  >&2 echo -e "${timestamp} [${level}] [$scriptname] ${message[*]}"
```

Comment:
Does shellcheck provide more info on the difference between `@` and `*`?

Reply:
Yup. Here's the error message:

> Argument mixes string and array. Use * or separate argument.

Reply:
See https://github.com/koalaman/shellcheck/wiki/SC2145

Reply:
Thx!